history/taskana-simplehistory-rest-spring/src/main/java/pro/taskana/simplehistory/rest/resource/TaskHistoryEventListResourceAssembler.java

```diff
@@ -4,56 +4,74 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
 import org.springframework.hateoas.IanaLinkRelations;
 import org.springframework.hateoas.Link;
 import org.springframework.hateoas.PagedModel.PageMetadata;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+import org.springframework.web.util.UriComponentsBuilder;
 
-import pro.taskana.resource.rest.AbstractRessourcesAssembler;
+import pro.taskana.common.rest.models.PagedResources;
 import pro.taskana.simplehistory.impl.HistoryEventImpl;
 import pro.taskana.simplehistory.rest.TaskHistoryEventController;
 
-/** Mapper to convert from a list of HistoryEventImpl to a TaskHistoryEventResource. */
-public class TaskHistoryEventListResourceAssembler extends AbstractRessourcesAssembler {
-
-  public TaskHistoryEventListResource toResources(
-      List<HistoryEventImpl> historyEvents, PageMetadata pageMetadata) {
+/**
+ * Mapper to convert from a list of HistoryEventImpl to a TaskHistoryEventResource.
+ */
+public class TaskHistoryEventListResourceAssembler {
```

Comment:
Wouldn't it be better to refactor this assembler like the workbasket assembler and get rid of the old paging style?

Reply:
Definitely. 
But this is part of the follow up ticket, where we're going to refactor the aspect and not only fixing the bug. 
Sadly this change is not related to the bug and was commited, because i cherry-picked the wrong commit.

Reply:
won't fix!
Not part of the Bugfix.